inc/pantheon-multisite-finalize.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/*
+ * If a site has multisite enabled, but has not had the final installation
+ * steps completed, alert the user and provide links.
+ */
+
+function pantheon_multisite_install_finalize_message() {
+	?>
+	<div class="notice notice-info is-dismissible">
+		<p><?php esc_html_e( 'Your WordPress Site Network is almost ready!', 'pantheon' ); ?></p>
+		<p><?php echo sprintf( __( 'Visit <a href="%s">Pantheon WordPress Site Network Configuration</a> for documentation on how to finalize configuration of your site network.', 'pantheon-systems' ), 'https://pantheon.io/docs/guides/multisite/config/#install-the-wordpress-site-network' ); ?></p>
```

Comment:
The textdomain here (`pantheon-systems`) is different than the textdomain on line 10 (`pantheon`). I see both in use in the codebase so there's no strong precedent, but I would probably lean to `pantheon` being the correct textdomain for the `mu-plugin` (or we change all references to `pantheon-mu-plugin`, but that's a separate issue). So, I would suggest updating this to:

```suggestion
		<p><?php echo sprintf( __( 'Visit <a href="%s">Pantheon WordPress Site Network Configuration</a> for documentation on how to finalize configuration of your site network.', 'pantheon' ), 'https://pantheon.io/docs/guides/multisite/config/#install-the-wordpress-site-network' ); ?></p>
```